auth/api/iam/api.go

```diff
@@ -453,15 +454,26 @@ func (r Wrapper) PresentationDefinition(ctx context.Context, request Presentatio
 		return nil, err
 	}
 
-	presentationDefinition, err := r.policyBackend.PresentationDefinition(ctx, *authorizer, request.Params.Scope)
+	definitions, err := r.policyBackend.PresentationDefinitions(ctx, *authorizer, request.Params.Scope)
 	if err != nil {
 		return nil, oauth.OAuth2Error{
 			Code:        oauth.InvalidScope,
 			Description: err.Error(),
 		}
 	}
 
-	return PresentationDefinition200JSONResponse(*presentationDefinition), nil
+	var definition *pe.PresentationDefinition
+	for _, def := range definitions {
+		if def.AudienceType == pe.AudienceTypeOrganization {
+			definition = &def.PresentationDefinition
```

Comment:
why does `PresentationDefinition()` only work for the scope intended for the organization wallet?